robolectric/src/test/java/org/robolectric/TestRunners.java

```diff
@@ -52,6 +60,46 @@ protected AndroidManifest createAppManifest(FsFile manifestFile, FsFile resDir,
     }
   }
 
+  public static class WithLayoutLibDefaults extends RobolectricTestRunner {
```

Comment:
Maybe we can avoid creating a dedicated test runner for this, better to enable this through a @Config value perhaps? @erd  thoughts?